### ISSUE TITLE
MacBooks, OS problem.

### ISSUE THREAD

**Issue by pagusek** [michaelyaakoby/testfun#13]
Hi Michael, 

at first I'd like to thank you for that library. It's awesome- testing was never so easy and finally it's pleasure to test dao / service methods :) 

Me and my friend came across one problem. I'm using Windows whereas my friend is using OS on Macbook. In my case tests are working perfectly, surpisingly the same code shared on repository is not working on Macbook. 

He figured out where the problem is. https://github.com/michaelyaakoby/testfun/blob/master/src/main/java/org/testfun/jee/runner/inject/ClassPathScanner.java, line 26. You are splitting the java.class.path with ";" but on the mac the entries are separated by ":". Is it possible that you fix that problem?

Bests,
Jacek.

**Comment by michaelyaakoby** [comment]
Thanks Jacek for the feedback and for finding this useful and for finding exactly where the problem is.

Its been a while since I wrote this (and since I've left Java in favor of Scala), however I think the fix is easy - should be possible to split by "[:;]" instead (split uses a regexp).
I've pushed a change but can't test it on Mac... 

Do you mind cloning the project, run "mvn install" then change your dependency to "0.13-SNAPSHOT" and see if it works for you and your friend.

If this works for both of you, I'll publish a new version.

Thanks,
Michael

**Comment by pagusek** [comment]
Thanks Michael, for your quick answer. We cloned the repository, checked and solution with array [:;] would solve the problem on Mac but caused on Windows, because on Windows that would be splited like this:

C
\Users\OEM\workspace\swalk\target\test-classes
C
\Users\OEM\workspace\swalk\target\classes

therefore the perfect proposed solution is:
String[] classPathRoots = System.getProperty("java.class.path",".").split(System.getProperty("path.separator"));

We ran mvn install both on Windows and Mac and tests are green. 

Bests,
Jacek.
